### ISSUE TITLE
Exercise 13.3.2: use similar code style as the original book

### ISSUE THREAD

**Issue by dongzhuoer** [jrnold/r4ds-exercise-solutions#129]
The book use `count(...) %>% filter(n > 1)`

```r
flights %>% 
  count(year, month, day, tailnum) %>% 
  filter(n > 1)
```

But you writes ` group_by(...) %>% filter(n() > 1) `

```r
fueleconomy::vehicles %>%
  group_by(id) %>%
  filter(n() > 1) 
```

**Comment by dongzhuoer** [comment]
By the way, your way have one advantage.

For `babynames::babynames`, `count(...) %>% filter(n > 1)` would result in error. 

Because `babynames::babynames` already contains a variable named `n`, so you have to write `count(...) %>% filter(nn > 1)`

**Comment by jrnold** [comment]
Thanks. I will change the code and add some explanation for the `babynames` example.

I'm not sure, but the `babynames` example may have been why I used that code originally (though it's more likely that I would write `group_by()` and `filter(n() ...)` anyways).

**Comment by jrnold** [comment]
Fixed in 392ba1503bbb7114b
